server/scripts/randomize-participants.ts

```diff
@@ -31,7 +32,7 @@ const generateParticipants = async (amount: number) => {
         preferredLocation: DEFAULT_PREF_LOCATION,
         interested: 'yes',
         crcClear: 'yes',
-        program: Math.floor(Math.random() * 2) === 0 && 'MHAW', //NOSONAR
+        program: Math.floor(Math.random() * 2) === 0 && Program.MHAW, // NOSONAR
```

Comment:
shoot, guess I didn't update this from one of my PR's, I think you're idea was to use `_.` something so it picks MHAW or HCA, if you don't mind?